### PR TITLE
database/sinkdb: document Op

### PR DESCRIPTION
**Pull request by kr**

### CODE REVIEW

**Comment by kr** [comment]
PTAL

File: database/sinkdb/op.go [hunk 1]
```diff
@@ -6,6 +6,18 @@ import (
...
 )
 
+// Op represents a change to the data store.
+// Each Op starts with conditions, boolean predicates over
+// existing stored data.
+// If all conditions return true, the Op is said to be satisfied.
+// It then results in zero or more effects,
+// mutations to apply to the data.
+// If there is more than one way to satisfy the conditions,
+// the effects may differ depending on how they are satisfied.
```
Comment (jbowens) [inline database/sinkdb/op.go]: This sentence confused me:
> If there is more than one way to satisfy the conditions, the effects may differ depending on how they are satisfied.

Should it be "If there is more than one way to apply the effects, the resulting state may differ depending on how they are applied."?

Comment (kr) [inline database/sinkdb/op.go]: Haha, this sentence was a little too aspirational. I'll just delete it for now.

**Comment by kr** [comment]
PTAL

**Comment by jbowens** [comment]
lgtm

**Pull request merged**